### PR TITLE
Accused transformer

### PR DESCRIPTION
**Pull request by tw-jeff-burroughs**
Screen Shot 2019-07-31 at 10 26 12 PM
Screen Shot 2019-07-31 at 10 26 21 PM
Screen Shot 2019-07-31 at 10 26 28 PM
Screen Shot 2019-08-01 at 11 03 55 AM

### CODE REVIEW

File: invisible_flow/entities/data_allegationcategory.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from attr import dataclass
+
+
+@dataclass
+class AllegationCategory:
```
Comment (terrencebunkley) [inline invisible_flow/entities/data_allegationcategory.py]: Can we match the name of the file to match the case of the Class name? ie `data_allegation_category` ?

File: invisible_flow/entities/data_officer.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+@dataclass
+class Officer:
+    birth_year: str
+    first_name: str
+    gender: str
+    last_name: str
+    middle_initial: str
+    race: str
+    rank: str
+    cr_id: str
```
Comment (terrencebunkley) [inline invisible_flow/entities/data_officer.py]: What's `cr_id` ?

File: invisible_flow/entities/data_policeunit.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from attr import dataclass
+
+
+@dataclass
+class PoliceUnit:
```
Comment (terrencebunkley) [inline invisible_flow/entities/data_policeunit.py]: Can we make this file name `data_police_unit`?

File: invisible_flow/entities/data_allegationcategory.py (hunk 1 above)
Comment (tw-jeff-burroughs) [reply invisible_flow/entities/data_allegationcategory.py]: We could do this, but it does not follow the convention already established by allegation, complainant, and investigator

File: invisible_flow/entities/data_officer.py (hunk 2 above)
Comment (tw-jeff-burroughs) [reply invisible_flow/entities/data_officer.py]: cr_id is the column that the LOG_NO column is converted to that allows us the ability to make queries across tables with the same original LOG_NO value

File: invisible_flow/entities/data_policeunit.py (hunk 3 above)
Comment (tw-jeff-burroughs) [reply invisible_flow/entities/data_policeunit.py]: We could do this, but it does not follow the convention already established by allegation, complainant, and investigator

**Pull request merged**